source/guides/network/wifi.rst

```diff
@@ -102,8 +104,119 @@ TUI (Text-based User Interface)
 
    .. figure:: /_figures/wifi/nmtui_5.png
 
+wpa_supplicant
+**************
+
+Make sure NetworkManager is either disabled or not installed before using
+wpa_supplicant.
+
+.. code-block:: bash
+
+   sudo systemctl stop NetworkManager.service
+
+#. Create a ``wpa_supplicant`` configuration directory.
+
+   .. code-block:: bash
+
+      sudo mkdir /etc/wpa_supplicant
+
+#. Create a minimal :file:`/etc/wpa_supplicant/wpa_supplicant.conf` and add the
+   following:
+
+   .. code-block:: bash
+
+      ctrl_interface=/run/wpa_supplicant
+      update_config=1
+
+#. Determine your wireless interface name.
+
+   .. code-block:: bash
+
+      iw dev
+
+#. Complete the configuration process as ``root``.
+
+   .. code-block:: bash
+
+      sudo su
+      wpa_supplicant -B -i $INTERFACE -c /etc/wpa_supplicant/wpa_supplicant.conf
+
+#. Use :command:`wpa_cli` (interactive mode) to scan for available networks.
+
+   .. code-block:: bash
+
+      > scan
+      OK
+      <3>CTRL-EVENT-SCAN-STARTED
+      <3>CTRL-EVENT-SCAN-RESULTS
+      > scan_results
+      bssid / frequency / signal level / flags / ssid
+      00:xx:xx:73:7b:46 5180 -55 [WPA2-PSK-CCMP][ESS] Network1
+      00:xx:xx:5d:d9:23 2412 -47 [RSN--CCMP][MESH] 1137e9
+      00:xx:xx:73:7b:43 2412 -49 [RSN--CCMP][MESH] 1137e9
+      00:xx:xx:5d:d9:25 2412 -47 [WPA2-PSK-CCMP][ESS] Network1
+      00:xx:xx:37:25:05 2412 -49 [WPA2-PSK-CCMP][ESS] Network1
+      00:xx:xx:73:7b:45 2412 -59 [WPA2-PSK-CCMP][ESS] Network1
+      00:xx:xx:83:fa:6a 2437 -57 [WPA-EAP-CCMP+TKIP][WPA2-EAP-CCMP+TKIP][ESS]
+      00:xx:xx:83:fa:70 5240 -76 [WPA2-EAP-CCMP][ESS] Network2
+      00:xx:xx:4f:e9:2c 2412 -67 [WPA2-PSK-CCMP][ESS][P2P] Printer
+      00:xx:xx:af:fe:3e 5765 -79 [WPA2-PSK-CCMP][ESS] Network3
+      00:xx:xx:e9:eb:29 2412 -76 [WPA2-PSK-CCMP][ESS] Network4
+      00:xx:xx:26:4a:b9 2412 -79 [WPA2-PSK-CCMP][ESS][P2P] Printer2
+      00:xx:xx:b9:0d:d4 2462 -79 [WPA2-PSK-CCMP][ESS] Network5
+
+#. Set up your network connection.
+
+   .. code-block:: bash
+
+      > add_network
+      0
+      > set_network 0 ssid "Network1"
+      OK
+      > set_network 0 psk "Network1Password"
+      OK
+      > enable_network 0
+      OK
+      <3>CTRL-EVENT-SCAN-STARTED
+      <3>CTRL-EVENT-SCAN-RESULTS
+      <3>SME: Trying to authenticate with 00:xx:xx:5d:d9:26 (SSID='Network1' freq=5180 MHz)
+      <3>Trying to associate with 00:xx:xx:5d:d9:26 (SSID='Network1' freq=5180 MHz)
+      <3>Associated with 00:xx:xx:5d:d9:26
+      <3>CTRL-EVENT-SUBNET-STATUS-UPDATE status=0
+      <3>WPA: Key negotiation completed with 00:xx:xx:5d:d9:26 [PTK=CCMP GTK=CCMP]
+      <3>CTRL-EVENT-CONNECTED - Connection to 00:xx:xx:5d:d9:26 completed [id=0 id_str=]
+
+#. Save the configuration, quit :command:`wpa_cli`, and log out of ``root``.
+
+   .. code-block:: bash
+
+      > save_config
+      OK
+      > quit
+      root@clr-live~ # exit
+
```

Comment:
Add a step here to enable the wpa_supplicant.service for future boots

One cool thing about the wpa_supplicant service is that it is a systemd template. So you can "pass" the interface name as the parameter to start a wpa_supplicant service targeting that interface specifically. 

This is the syntax: `systemctl enable --now wpa_supplicant@wlp4s0.service` which expects to read a configuration at `/etc/wpa_supplicant/wpa_supplicant-wlp4s0.conf`